README.md

```diff
@@ -65,6 +65,36 @@ Web context path to use, such as "/rundeck". http://host.domain:port/server_web_
 #####`ssl_enabled`
 Enable ssl for the Rundeck web application.
 
+#####`ssl_keyfile`
+#####`ssl_certfile`
+If ssl enabled you need set up this. You need provide the crt and key files in your profile oder role manifest.
```

Comment:
This sentence needs a little more work in order to be clear to users. 

Let's start with something like this:

"If `ssl_enabled` is True, you must supply this parameter. It is recommended that you provide the `.crt` and `.key` files separately via other means, such as a role or profile manifest."